src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -2049,6 +2169,18 @@ type BookingInfo {
     dropOffMessage: String
 }
 
+input OptimizationCriteria @oneOf {
+    type: OptimizationType
+    triangle: InputTriangle
+}
+
+enum OptimizationType {
+    QUICK
```

Comment:
We can debate if we want to rename this to something like "lowest distance" because it can route you, for example, through a road with terrible cobblestones, so you will not be fast.

Reply:
SHORTEST?

Reply:
i like SHORTEST

Reply:
but on the other hand, pretty much every other word also has problems